qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/parser/expression/NamespacePart.java

```diff
@@ -36,6 +41,27 @@ public NamespacePart(int start, int end) {
 	public PartKind getPartKind() {
 		return PartKind.Namespace;
 	}
+	
+	@Override
+	public int getStartName() {
+		if (startName != -1) {
+			return startName;
+		}
+		startName = super.getStartName();
+		Parameter parameter = getOwnerParameter();
+		if (parameter != null) {
+			Section section = parameter.getOwnerSection();
+			if (section != null && section.getSectionKind() == SectionKind.IF) {
+				String text = getOwnerTemplate().getText();
+				if (text.charAt(startName) == '!') {
+					// ex : !inject:
+					// !inject --> inject
+					startName++;

```

Comment:
You can have a space after the `!` and it will still work.

eg.

```qute-html
{#if  ! inject:vertxRequest.getParam('sandwich')??}
no sandwich ;-;
{/}
```

Reply:
Could we see this usecase in an another PR because this usecase is used in renarde and I would like to integrate this fix in the release. We have supported this usacse for object part before and I copy/paste this code for namespace part.

Reply:
Okay. I opened https://github.com/redhat-developer/quarkus-ls/issues/839 to address this issue. It doesn't make sense to do the same fix for object part, since `{!` marks the start of a comment.